docs/01-prerequisites.md

```diff
@@ -1,167 +1,11 @@
-The provisioning host you are using must have the AWS command line tools
-(CLI) installed. See
-https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html
-
-After installing the AWS CLI, it must be configured for the AWS account and
-IAM role. This assumes we are using the `openshift4-beta-admin` IAM user
-mentioned earlier:
-
-```console
-$ aws configure --profile=openshift4-beta-admin
-AWS Access Key ID [None]: <for the AWS role openshift4-beta-admin>
-AWS Secret Access Key [None]: <for the AWS role openshift4-beta-admin>
-Default region name [None]: <the AWS region to deploy the beta environment>
-Default output format [None]: text
-```
-
-Configure everything to use this new profile
-
-```console
-$ export AWS_PROFILE=openshift4-beta-admin
-```
-
-## SSH Keys
-
-The installer will interactively prompt for an SSH public key if it
-finds one in `~/.ssh/` that ends with `.pub`, and you can use the
-arrow keys to select the one you wish to use.  If no `.pub` file is
-found, the installer will not prompt for public key and will install
-the cluster without SSH access.
-
-If you install a cluster without initially selecting an SSH public
-key, you can [configure these later using the machine-config
-operator][machine-config-daemon-ssh-keys].
-
-## OpenShift Installer
-
-1. Visit https://try.openshift.com with your web browser
-1. Login with the Red Hat Account that is in the Developer Program
-1. Following that, you will be re-directed to
-    https://cloud.openshift.com/clusters/install Scroll down to the section
-    called “Download the installer”
-1. Download the installer; You will be re-directed to
-    https://github.com/openshift/installer/releases where you can find the
-    appropriate binary for your host (openshift-install-linux-amd64 or
-    openshift-install-darwin-amd64)
-1. Once the download is complete, rename the installer and make it
-    executable. For example:
-    ```
-    mv openshift-install-darwin-amd64 openshift-install
-    chmod +x openshift-install
-    ```
-1. Download the Pull Secret from https://cloud.openshift.com/clusters/install
-  and make sure you have its contents available to you
-
-## OpenShift CLI
-On https://cloud.openshift.com/clusters/install, scroll down to the section
-called “Access your new cluster!” and click the button "Download Command-line
-Tools". Download the appropriate CLI for your platform, extract it, make sure
-it is executable, and make sure that it is in your `PATH`.
-
-For example, for Linux or MacOS:
-
-~~~bash
-$ wget <path to oc tarball>
-(...)
-$ tar -zxvf oc.tar.gz
-oc
-
-$ export PATH=$PATH:<path to your oc location>
-$ oc version
-oc v4.0.0-0.148.0
-kubernetes v1.11.0+8868a98a7b
-features: Basic-Auth GSSAPI Kerberos SPNEGO
-~~~
-
-# Problems?
-If at any point you run into issues while exploring OpenShift 4, see the
-[troubleshooting](99-troubleshooting.md) section.
-
-
-Next: [Installing the Cluster](02-install.md)
+# Configuring an AWS Account
+Refer to the [AWS
```

Comment:
```suggestion
Refer to [Configuring an AWS
```

Reply:
Is this a docs stylistic thing?

Reply:
I think it sounds like the link should be in the AWS doc, not the OCP doc about AWS.